MekHQ/src/mekhq/campaign/mission/enums/CombatRole.java

```diff
@@ -125,19 +143,18 @@ public static CombatRole parseFromString(final String text) {
             case "IN_RESERVE", "UNASSIGNED" -> RESERVE;
             default -> {
                 MMLogger.create(CombatRole.class)
-                    .warn(String.format("Unable to parse %s into an CombatRole. Returning RESERVE.",
-                        text));
+                      .warn(String.format("Unable to parse %s into an CombatRole. Returning RESERVE.", text));
```

Comment:
`.warn(String.format("Unable to parse %s into an CombatRole. Returning RESERVE.", text))`

can be `.warn("Unable to parse {} into an CombatRole. Returning RESERVE.", text)` and save some characters and overhead.

I know this is just the formatter kicking in, but just letting you know.